cmd/agent/subcommands/snmp/command.go

```diff
@@ -232,7 +232,7 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 	snmpScanCmd.Flags().BoolVar(&connParams.UseUnconnectedUDPSocket, "use-unconnected-udp-socket", defaultUseUnconnectedUDPSocket, "If specified, changes net connection to be unconnected UDP socket")
 
 	// This command does nothing until the backend supports it, so it isn't enabled yet.
-	// snmpCmd.AddCommand(snmpScanCmd)
+	snmpCmd.AddCommand(snmpScanCmd)
```

Comment:
Is there any way to hide this from the cobra-generated help, so that users won't think it does something when it doesn't yet?

Reply:
I'm not sure we can hide subcommands form the help
I can keep this PR opened until we are ready to launch this

Reply:
I think you can pass `Hidden: true` when creating `snmpScanCmd` here
https://github.com/DataDog/datadog-agent/blob/1dc09461c4a89b88d97b01744446f0dc80621409/cmd/agent/subcommands/snmp/command.go#L178

Reply:
good catch ! that does the trick
added in [b4b5bf9](https://github.com/DataDog/datadog-agent/pull/29553/commits/b4b5bf93d5ee33d3beff99d715d75a0a3f71806d)